CHANGELOG.md

```diff
@@ -0,0 +1,118 @@
+# Changelog
```

Comment:
Should we remove the changelog, since the past changes are not relevant for this repo?